src/bp-templates/bp-nouveau/js/buddypress-messages.js

```diff
@@ -2702,6 +2702,7 @@ window.bp = window.bp || {};
 						}
 					);
 					content = bp.Nouveau.Messages.mediumEditor.getContent();
+					content = content.replace(/&nbsp;/g,'').trim();
```

Comment:
@buddy4kp 
`content` variable is from 2 either cases. 1 is from tinyMCE condition above and other is here from mediumEditor You will need to put this outside conditions.

Reply:
@mehulkaklotar 
I have applied below changes as per suggestion.
 - Put the `content` variable outside the condition to apply in both cases tinyMCE and mediumEditor
 - Applied the same changes in `sendMessage` for New Message form